e2e/test/scenarios/native/native.cy.spec.js

```diff
@@ -459,6 +459,49 @@ describe("no native access", { tags: ["@external", "@quarantine"] }, () => {
   });
 });
 
+describe("scenarios > question > native", { tags: "@mongo" }, () => {
+  const MONGO_DB_NAME = "QA Mongo4";
+
+  before(() => {
```

Comment:
Let's either use `beforeEach` or let's lose the `describe block` and put everything inside a single `it()` block.